scripts/run-local-tests.sh

```diff
@@ -26,7 +26,7 @@ cleanup() {
 trap cleanup EXIT
 
 # Load data for tests
-docker exec veda.db /tmp/scripts/bin/load-data.sh
+docker exec veda.loadtestdata /tmp/scripts/bin/load-data.sh
```

Comment:
I guess we don't need this anymore but I can't decide if it is still worth having here as pseudo documentation